dev/bots/suite_runners/run_android_engine_tests.dart

```diff
@@ -77,7 +77,7 @@ Future<void> runAndroidEngineTests({required ImpellerBackend impellerBackend}) a
       final String? stdout = result.flattenedStdout;
       if (stdout == null) {
         foundError(<String>['No stdout produced.']);
-        continue;
+        return;
```

Comment:
I think I had this as a continue so that hypothetically other tests ran, but I am not sure it was worth it.

If this was intentional feel free to ignore.

Reply:
Talked offline, this is pulled into a closure now so its equivalent code.